docs/docsite/rst/dev_guide/testing_units_modules.rst

```diff
@@ -459,7 +460,7 @@ mock for :meth:`Ansible.get_bin_path`::
                 mock_run_command.return_value = rc, stdout, stderr  # successful execution
 
                 with self.assertRaises(AnsibleExitJson) as result:
-                    my_module.main()
+                    self.module.main()
```

Comment:
This change, and the one above it, should be unnecessary, since `my_module` is in scope from the import above:

https://github.com/ansible/ansible/pull/65274/files#diff-51e739f8c9305f3df59b521475f34f1fR392